enumerable.rb

```diff
@@ -1,4 +1,4 @@
-module My_Enumerable
+module MyEnumerable
```

Comment:
- [ ] I think you have missed changing the name of the module name in `my_list.rb`. Please update it.